.env

```diff
@@ -1 +1 @@
-ESLINT_NO_DEV_ERRORS=true
+VITE_API_KEY=AIzaSyD0ftUETmycY3ljtwDjD7cyvdAeEKENdI0
```

Comment:
bad practice have env on GH
it should be hidden
you could have env.example without value